packages/terra-navigation-prompt/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ Changelog
 
 Unreleased
 ----------
+### Changed
+* Changed how 'navigation-prompt' was consuming 'terra-notifiaction-dialog' to deal with it's updated props.
```

Comment:
~it's~ -> its

Reply:
Fixed here https://github.com/cerner/terra-framework/pull/899/commits/9fb621a134f79a08d05db32ebf10603b3027fc3a#diff-22332093e32474b48af82c8669855bd2R7